webcompat/webhooks/helpers.py

```diff
@@ -39,6 +39,7 @@ def parse_and_set_label(body, issue_number):
         browser = match_list.groups(0)[1].lower()
         dash_browser = '-'.join(browser.split())
         set_label('browser-' + dash_browser, issue_number)
+        set_label('status-needstriage', issue_number)
```

Comment:
I think we shouldn't add this label here in `parse_and_set_label` -- it doesn't match the description of the method (or the method name). We could either make a new helper method like `add_default_label`, or just call `set_label` directly from __init__.py (after `parse_and_set_label`, with a comment explaining what's happening).